var/spack/repos/builtin/packages/mapnik/package.py

```diff
@@ -0,0 +1,55 @@
+# Copyright 2013-2019 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class Mapnik(AutotoolsPackage):
+    """
+    mapnik combines pixel-perfect image output with lightning-fast
+    cartographic algorithms, and exposes interfaces in C++, Python, and Node
+    """
+
+    homepage = "https://mapnik.org/"
+    url      = "https://github.com/mapnik/mapnik/releases/download/v3.0.22/mapnik-v3.0.22.tar.bz2"
+
+    version('3.0.22', sha256='930612ad9e604b6a29b9cea1bc1de85cf7cf2b2b8211f57ec8b6b94463128ab9')
+
+    depends_on('python', type=('build', 'run'))
+    # Build fails with boost@1.70
+    depends_on('boost@:1.69.0+regex+filesystem+system+icu+program_options+python cxxstd=11')
+    depends_on('icu4c')
+    depends_on('zlib')
+    depends_on('freetype')
+    depends_on('libxml2')
+    depends_on('harfbuzz')
+    depends_on('libpng')
+    depends_on('libjpeg')
+    depends_on('libtiff')
+    depends_on('proj')
+    depends_on('cairo')
+    depends_on('postgresql', type=('build', 'link', 'run'))
+    depends_on('gdal', type=('build', 'link', 'run'))
+    depends_on('sqlite+rtree', type=('build', 'link', 'run'))
+    depends_on('libwebp')
+    depends_on('python', type='build')
```

Comment:
Redundant

Reply:
It is actually a Scons package, so `configure` is just a bash script running python. Should we rely on python that is enabling Spack to run Scons stuff or should we have a  build only python dependency as above?

Reply:
Hmm, in that case I wouldn't call it an `AutotoolsPackage`, either `Package` or `SConsPackage`.

By redundant I meant that you already have a build/run dep on Python above.

Reply:
> Hmm, in that case I wouldn't call it an `AutotoolsPackage`, either `Package` or `SConsPackage`.

When I created the package, spack thought it is an autotools package due to configure I think.
 
> By redundant I meant that you already have a build/run dep on Python above.

Didn't notice that! Thanks

Reply:
> When I created the package, spack thought it is an autotools package due to configure I think.

Yep, Spack isn't perfect, sometimes you have to help it along a little.

Reply:
On first attempt, not building as a sconspackage:
```
KeyError: 'BOOST_APPEND':
  File "/mnt/local/cache/sbulut/spack-stage/mapnik-3.0.22-rgj4jxkwsz5e3km6idsce3v25gqbbgyz/spack-src/SConstruct", line 1561:
    if not conf.CheckLibWithHeader('boost_%s%s' % (libinfo[0],env['BOOST_APPEND']), libinfo[1], 'C++'):
  File "/disk/software/lib/hpcpm/spack/opt/spack/linux-centos7-x86_64/gcc-7.4.0/scons-3.1.0-s3e6sumdvy5xglpbptajqifviqrlklq2/lib/scons/SCons/Environment.py", line 410:
    return self._dict[key]
```

Reply:
That error just means you need to set the `BOOST_APPEND` environment variable.

Reply:
set some environment variables still getting a similar error. Let's merge this PR as it is, and I will work on this further in the following weeks.